### PR TITLE
Change NoOps to SQLite across all project folders

### PR DESCRIPTION
**Pull request by tpmccallum**
Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [x] The `title, `template`, and `date` are all set
- [x] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [x] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [x] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)
- [x] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)
- [x] Headings are using Title Case
- [x] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [x] Have tested with `npm run test` and resolved all errors
- [x] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

### CODE REVIEW

File: content/cloud/changelog/cloud-noops-sql-db.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ type= "changelog_post"
 
 ---
 
-Spin developers can now persist relational data generated by their Spin application in a database that is provisioned and managed entirely by Fermyon Cloud on their behalf. No need to worry about database operations, connection strings, or resource management with Fermyon Cloud's NoOps SQLite Database; we've got you covered.
+Spin developers can now persist relational data generated by their Spin application in a database that is provisioned and managed entirely by Fermyon Cloud on their behalf. No need to worry about database operations, connection strings, or resource management with Fermyon Cloud's SQLite Database; we've got you covered.
```
Comment (macolso) [inline content/cloud/changelog/cloud-noops-sql-db.md:11]: Can we add a callout to help handhold users through the transition?

```suggestion
Spin developers can now persist relational data generated by their Spin application in a database that is provisioned and managed entirely by Fermyon Cloud on their behalf. No need to worry about database operations, connection strings, or resource management with Fermyon Cloud's SQLite Database; we've got you covered.

>> Note: Fermyon Cloud NoOps SQL Database has been renamed to Fermyon Cloud SQLite Database. Your workload is not impacted by this name change, and no actions are required on your end. 
```

**Review by macolso — COMMENTED** [review]
Looks great, thank you @tpmccallum! I added a suggestion to add a brief callout to help user through the name transition.

**Comment by itowlson** [comment]
Wait what's this. are we meant to be changing this in the cloud plugin and stuff as well?

**Review by technosophos — APPROVED** [review]
Approve, with @macolso 's suggested addition

File: content/cloud/faq.md [hunk 2]
```diff
@@ -42,8 +42,8 @@ The following are the quota for users based, stratified by the three [Subscripti
 | Key value maximum keys* | 1,024 | 1,024 | Contact us
 | Key value store count | 5 | 100 | Contact us
 | Key value storage size (GB) | 1 | 2 | Contact us
-| SQLite database count | 1 | 1 |  Contact us
-| SQLite database size (GB) | 1 | 2 | Contact us
+| SQLite Database count | 1 | 1 |  Contact us
```
Comment (itowlson) [inline content/cloud/faq.md:45]: I don't think this is the brand name, just "database" in its "the concept that Fermyon Cloud SQLite Database implements" which is why it wasn't NoOps before

```suggestion
| SQLite database count | 1 | 1 |  Contact us
```

File: content/cloud/faq.md [hunk 3]
```diff
@@ -42,8 +42,8 @@ The following are the quota for users based, stratified by the three [Subscripti
...
+| SQLite Database size (GB) | 1 | 2 | Contact us
```
Comment (itowlson) [inline content/cloud/faq.md:46]:
```suggestion
| SQLite database size (GB) | 1 | 2 | Contact us
```

File: content/cloud/linking-applications-to-resources-using-labels.md [hunk 4]
```diff
@@ -11,13 +11,13 @@ url = "https://github.com/fermyon/developer/blob/main/content/cloud/linking-appl
...
+Many Spin applications require data storage between invocations. Fermyon Cloud offers two methods for this: key-value storage and SQLite Databases. However, the relationship between applications and databases isn't strictly one-to-one. A single database might be used by multiple applications, while an application could utilize several databases. There might even be a need to maintain a database that isn't utilized by any applications, perhaps for analytics or compliance reasons. All of these scenarios can occur as applications are independently deployed, upgraded, and retired, without any particular application needing to be aware of how others might be using the same databases.
```
Comment (itowlson) [inline content/cloud/linking-applications-to-resources-using-labels.md:14]: again, this was not previously captialised - next to "key-value storage" it looks weird to me but I don't know what the official branding guidelines say _grin_

File: content/cloud/noops-sql-db.md [hunk 5]
```diff
@@ -144,5 +144,5 @@ Database "inspirational-pig" deleted
 
 ## Next Steps
 
-* For support or feature feedback, please join our [Discord channel #no-ops-db-beta](https://discord.gg/XxkZvsya). This is where you can provide feedback and ask for assistance from the Fermyon Cloud team on all things NoOps SQL Database. 
-* Visit [FAQ](/cloud/faq) for frequently asked questions
+* For support or feature feedback, please join our [Discord channel #no-ops-db-beta](https://discord.gg/XxkZvsya). This is the place where you can provide feedback and request assistance from the Fermyon Cloud team regarding all matters related to using SQLite Databases in Spin and Fermyon Cloud.
```
Comment (itowlson) [inline content/cloud/noops-sql-db.md:147]: How do you feel about all the extra words here?  To me "all things SQLite Database" feels more concise than "all matters related to using SQLite Databases in Spin and Fermyon Cloud."  Is the idea that non-English speakers might find the former too idiomatic?

File: content/spin/dynamic-configuration.md [hunk 6]
```diff
@@ -141,7 +141,7 @@ You can supply runtime configuration by providing a value for the `--runtime-con
 
 ### Key Value Store Runtime Configuration
 
-Spin provides built-in key-value storage. This storage is backed by an SQLite database embedded in Spin by default. However, the Spin runtime configuration file (runtime-config.toml) can be updated to not only modify the SQLite configuration but also choose to use a different backing store. The available store options are the embedded SQLite database, an external Redis database or Azure CosmosDB.
+Spin provides built-in key-value storage. This storage is backed by an SQLite Database embedded in Spin by default. However, the Spin runtime configuration file (runtime-config.toml) can be updated to not only modify the SQLite configuration but also choose to use a different backing store. The available store options are the embedded SQLite Database, an external Redis database or Azure CosmosDB.
```
Comment (itowlson) [inline content/spin/dynamic-configuration.md:144]: This is definitely _not_ the Cloud brand name, it's just an ordinary non-proper-named SQLite database on your local disk.

```suggestion
Spin provides built-in key-value storage. This storage is backed by an SQLite database embedded in Spin by default. However, the Spin runtime configuration file (runtime-config.toml) can be updated to not only modify the SQLite configuration but also choose to use a different backing store. The available store options are the embedded SQLite database, an external Redis database or Azure CosmosDB.
```

File: content/spin/dynamic-configuration.md [hunk 7]
```diff
@@ -184,9 +184,9 @@ You must individually grant each component access to the stores that it needs to
 
 ### SQLite Storage Runtime Configuration
 
-Spin provides built-in SQLite storage. By default, this is backed by a database that Spin creates for you underneath your application directory (in the `.spin` subdirectory). However, you can use the Spin runtime configuration file (`runtime-config.toml`) to add and customize SQLite databases.
+Spin provides built-in SQLite storage. By default, this is backed by a database that Spin creates for you underneath your application directory (in the `.spin` subdirectory). However, you can use the Spin runtime configuration file (`runtime-config.toml`) to add and customize SQLite Databases.
```
Comment (itowlson) [inline content/spin/dynamic-configuration.md:187]: Same
```suggestion
Spin provides built-in SQLite storage. By default, this is backed by a database that Spin creates for you underneath your application directory (in the `.spin` subdirectory). However, you can use the Spin runtime configuration file (`runtime-config.toml`) to add and customize SQLite databases.
```

File: content/spin/dynamic-configuration.md [hunk 8]
```diff
@@ -184,9 +184,9 @@ You must individually grant each component access to the stores that it needs to
...
 
-The following example `runtime-config.toml` tells Spin to map the `default` database to an SQLite database elsewhere in the file system:
+The following example `runtime-config.toml` tells Spin to map the `default` database to an SQLite Database elsewhere in the file system:
```
Comment (itowlson) [inline content/spin/dynamic-configuration.md:189]:
```suggestion
The following example `runtime-config.toml` tells Spin to map the `default` database to an SQLite database elsewhere in the file system:
```

File: content/spin/rdbms-storage.md [hunk 9]
```diff
@@ -10,7 +10,7 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/rdbms-storage
 
 Spin provides two interfaces for relational (SQL) databases:
 
-* A built-in [SQLite database](./sqlite-api-guide), which is always available and requires no management on your part.
+* A built-in [SQLite Database](./sqlite-api-guide), which is always available and requires no management on your part.
```
Comment (itowlson) [inline content/spin/rdbms-storage.md:13]:
```suggestion
* A built-in [SQLite database](./sqlite-api-guide), which is always available and requires no management on your part.
```

File: content/spin/see-what-people-have-built-with-spin.md [hunk 10]
```diff
@@ -64,9 +65,9 @@ Chances are whatever you are building will want to talk to other endpoints on th
 
 - [Accessing external APIs](https://www.fermyon.com/blog/spin-rest-apis)
 
-## NoOps SQLite Storage Using Javascript
+## SQLite Storage Using Javascript
 
-Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides a NoOps SQLite Storage example (using Javascript).
+Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides a SQLite Storage example (using Javascript).
```
Comment (itowlson) [inline content/spin/see-what-people-have-built-with-spin.md]:
```suggestion
Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides a SQLite Database example (using JavaScript).
```

File: content/spin/sqlite-api-guide.md [hunk 11]
```diff
@@ -12,24 +12,24 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/sqlite-api-gu
 - [Custom SQLite Databases](#custom-sqlite-databases)
   - [Granting Access to Custom SQLite Databases](#granting-access-to-custom-sqlite-databases)
 
-Spin provides an interface for you to persist data in an SQLite database managed by Spin. This database allows Spin developers to persist relational data across application invocations.
+Spin provides an interface for you to persist data in an SQLite Database managed by Spin. This database allows Spin developers to persist relational data across application invocations.
```
Comment (itowlson) [inline content/spin/sqlite-api-guide.md:15]:
```suggestion
Spin provides an interface for you to persist data in an SQLite database managed by Spin. This database allows Spin developers to persist relational data across application invocations.
```

File: content/spin/sqlite-api-guide.md [hunk 12]
```diff
@@ -12,24 +12,24 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/sqlite-api-gu
...
 
-{{ details "Why do I need a Spin interface? Why can't I just use my own external database?" "You can absolutely still use your own external database either with the [MySQL or Postgres APIs](./rdbms-storage). However, if you're interested in quick, local relational storage without any infrastructure set-up then Spin's SQLite database is a great option." }}
+{{ details "Why do I need a Spin interface? Why can't I just use my own external database?" "You can absolutely still use your own external database either with the [MySQL or Postgres APIs](./rdbms-storage). However, if you're interested in quick, local relational storage without any infrastructure set-up then Spin's SQLite Database is a great option." }}
```
Comment (itowlson) [inline content/spin/sqlite-api-guide.md:17]:
```suggestion
{{ details "Why do I need a Spin interface? Why can't I just use my own external database?" "You can absolutely still use your own external database either with the [MySQL or Postgres APIs](./rdbms-storage). However, if you're interested in quick, local relational storage without any infrastructure set-up then Spin's SQLite database is a great option." }}
```

File: content/spin/sqlite-api-guide.md [hunk 13]
```diff
@@ -12,24 +12,24 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/sqlite-api-gu
...
 
 ## Granting SQLite Database Permissions to Components
 
-By default, a given component of an app will not have access to any SQLite databases. Access must be granted specifically to each component via the component manifest.  For example, a component could be given access to the default store using:
+By default, a given component of an app will not have access to any SQLite Databases. Access must be granted specifically to each component via the component manifest.  For example, a component could be given access to the default store using:
```
Comment (itowlson) [inline content/spin/sqlite-api-guide.md:21]:
```suggestion
By default, a given component of an app will not have access to any SQLite databases. Access must be granted specifically to each component via the component manifest.  For example, a component could be given access to the default store using:
```

File: content/spin/sqlite-api-guide.md [hunk 14]
```diff
@@ -12,24 +12,24 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/sqlite-api-gu
...
 sqlite_databases = ["default"]
 ```
 
-> Note: To deploy your Database application to Fermyon Cloud using `spin cloud deploy`, see the [NoOps SQL Database](/cloud/noops-sql-db#accessing-private-beta) section in the documentation. It covers signing up for the private beta and setting up your Cloud database tables and initial data.
+> Note: To deploy your Database application to Fermyon Cloud using `spin cloud deploy`, see the [SQLite Database](/cloud/noops-sql-db#accessing-private-beta) section in the documentation. It covers signing up for the private beta and setting up your Cloud database tables and initial data.
 
 ## Using SQLite Storage From Applications
 
-The Spin SDK surfaces the Spin SQLite database interface to your language.
+The Spin SDK surfaces the Spin SQLite Database interface to your language.
```
Comment (itowlson) [inline content/spin/sqlite-api-guide.md:32]:
```suggestion
The Spin SDK surfaces the Spin SQLite database interface to your language.
```

File: content/spin/sqlite-api-guide.md [hunk 15]
```diff
@@ -244,7 +244,7 @@ Spin defines a database named `"default"` and provides automatic backing storage
 
 ### Granting Access to Custom SQLite Databases
 
-As mentioned above, by default, a given component of an app will not have access to any SQLite databases. Access must be granted specifically to each component via the component manifest, using the `component.sqlite_databases` field in the manifest.
+As mentioned above, by default, a given component of an app will not have access to any SQLite Databases. Access must be granted specifically to each component via the component manifest, using the `component.sqlite_databases` field in the manifest.
```
Comment (itowlson) [inline content/spin/sqlite-api-guide.md:247]:
```suggestion
As mentioned above, by default, a given component of an app will not have access to any SQLite databases. Access must be granted specifically to each component via the component manifest, using the `component.sqlite_databases` field in the manifest.
```

File: content/spin/v2/dynamic-configuration.md [hunk 16]
```diff
@@ -141,7 +141,7 @@ You can supply runtime configuration by providing a value for the `--runtime-con
 
 ### Key Value Store Runtime Configuration
 
-Spin provides built-in key-value storage. This storage is backed by an SQLite database embedded in Spin by default. However, the Spin runtime configuration file (runtime-config.toml) can be updated to not only modify the SQLite configuration but also choose to use a different backing store. The available store options are the embedded SQLite database, an external Redis database or Azure CosmosDB.
+Spin provides built-in key-value storage. This storage is backed by an SQLite Database embedded in Spin by default. However, the Spin runtime configuration file (runtime-config.toml) can be updated to not only modify the SQLite configuration but also choose to use a different backing store. The available store options are the embedded SQLite Database, an external Redis database or Azure CosmosDB.
```
Comment (itowlson) [inline content/spin/v2/dynamic-configuration.md:144]: I'm guessing the same ones as in the previous Spin docs are also going to be in the `v2` directory - take this comment as standing in repeating all the suggestions on the v1 docs _grin_

```suggestion
Spin provides built-in key-value storage. This storage is backed by an SQLite database embedded in Spin by default. However, the Spin runtime configuration file (runtime-config.toml) can be updated to not only modify the SQLite configuration but also choose to use a different backing store. The available store options are the embedded SQLite database, an external Redis database or Azure CosmosDB.
```

**Review by itowlson — CHANGES_REQUESTED** [review]
I'm afraid it's gone a bit awry on the Spin side.  We should not capitalise "database" except in the marketing context; e.g. when we mention that local KV is backed by a SQLite database, that's _not_ a Fermyon Cloud Capitalised Database, it's just a normal improper noun.

Other than those, looks good - thanks!

File: content/cloud/linking-applications-to-resources-using-labels.md (hunk 4 above)
Comment (tpmccallum) [inline content/cloud/linking-applications-to-resources-using-labels.md:14]:
```suggestion
Many Spin applications require data storage between invocations. Fermyon Cloud offers two methods for this: key-value storage and SQLite databases. However, the relationship between applications and databases isn't strictly one-to-one. A single database might be used by multiple applications, while an application could utilize several databases. There might even be a need to maintain a database that isn't utilized by any applications, perhaps for analytics or compliance reasons. All of these scenarios can occur as applications are independently deployed, upgraded, and retired, without any particular application needing to be aware of how others might be using the same databases.
```

File: content/cloud/noops-sql-db.md (hunk 5 above)
Comment (tpmccallum) [inline content/cloud/noops-sql-db.md:147]:
```suggestion
* For support or feature feedback, please join our [Discord channel #no-ops-db-beta](https://discord.gg/XxkZvsya). This is the place where you can provide feedback and request assistance from the Fermyon Cloud team regarding all things SQLite Database.
```

**Review by tpmccallum — COMMENTED** [review]
Couple of capitalization updates.

File: content/spin/see-what-people-have-built-with-spin.md (hunk 10 above)
Comment (tpmccallum) [inline content/spin/see-what-people-have-built-with-spin.md]:
```suggestion
Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides an SQLite Database example (using JavaScript).
```

**Review by tpmccallum — COMMENTED** [review]
Respond to review.

File: content/spin/see-what-people-have-built-with-spin.md (hunk 10 above)
Comment (tpmccallum) [reply content/spin/see-what-people-have-built-with-spin.md]: This is outdated it seems.

Comment (tpmccallum) [inline content/spin/see-what-people-have-built-with-spin.md]:
```suggestion
Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides an SQLite Database example (using JavaScript).
```

**Review by tpmccallum — COMMENTED** [review]
Review

File: content/spin/see-what-people-have-built-with-spin.md [hunk 17]
```diff
@@ -64,9 +65,9 @@ Chances are whatever you are building will want to talk to other endpoints on th
 
 - [Accessing external APIs](https://www.fermyon.com/blog/spin-rest-apis)
 
-## NoOps SQLite Storage Using Javascript
+## SQLite Storage Using Javascript
 
-Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides a NoOps SQLite Storage example (using Javascript).
+Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides an SQLite Storage example (using Javascript).
```
Comment (tpmccallum) [inline content/spin/see-what-people-have-built-with-spin.md:70]:
```suggestion
Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides an SQLite database example (using Javascript).
```

File: content/spin/v2/see-what-people-have-built-with-spin.md [hunk 18]
```diff
@@ -64,9 +64,9 @@ Chances are whatever you are building will want to talk to other endpoints on th
 
 - [Accessing external APIs](https://www.fermyon.com/blog/spin-rest-apis)
 
-## NoOps SQLite Storage Using Javascript
+## SQLiteite Storage Using Javascript
 
-Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides a NoOps SQLite Storage example (using Javascript).
+Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides an SQLiteite Storage example (using Javascript).
```
Comment (tpmccallum) [inline content/spin/v2/see-what-people-have-built-with-spin.md:69]:
```suggestion
Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides an SQLite database example (using Javascript).
```

**Review by tpmccallum — COMMENTED** [review]
Trying to resolve outdated suggestion issue

**Review by tpmccallum — COMMENTED** [review]
Typo

**Comment by tpmccallum** [comment]
Thank @itowlson 
Happy with this one if you are; thanks for the extra suggestions.

**Review by itowlson — APPROVED** [review]

File: content/spin/cli-reference.md [hunk 19]
```diff
@@ -4382,7 +4382,7 @@ TRIGGER OPTIONS:
             [env: RUNTIME_CONFIG_FILE=]
 
         --sqlite <SQLITE_STATEMENTS>
-            Run a SQLite statement such as a migration against the default database. To run from a
+            Run an SQLite statement such as a migration against the default database. To run from a
```
Comment (tpmccallum) [inline content/spin/cli-reference.md:4385]:
```suggestion
            Run a SQLite statement such as a migration against the default database. To run from a
```

File: content/spin/cli-reference.md [hunk 20]
```diff
@@ -4306,7 +4306,7 @@ TRIGGER OPTIONS:
...
```
Comment (tpmccallum) [inline content/spin/cli-reference.md:4309]:
```suggestion
            Run a SQLite statement such as a migration against the default database. To run from a
```

**Review by tpmccallum — COMMENTED** [review]
Only change written text (not terminal output)

**Review by itowlson — APPROVED** [review]

**Pull request merged**